### ISSUE TITLE
node.js 22.11.0

### ISSUE THREAD

**Issue by vzlgdu** [arachnetech/homebridge-mqttthing#689]
Hi, I'm receiving this message:

The plugin "homebridge-mqttthing" requires Node.js version of ^16.10.0 || ^18.12.0 || ^20.10.0 which does not satisfy the current Node.js version of v22.11.0. You may need to upgrade your installation of Node.js

What should I do if my node.js is higher than 20.10.0?

Will be your plugin updated?

Thanks

**Comment by vzlgdu** [comment]
Is there someone that has installed node.js 22.11.0?

**Comment by GithHobby** [comment]
Actually I'm using v22.12.0 without problems

**Comment by GithHobby** [comment]
yes, there is the message but maybe is because v22 is not 16, 18 or 20.
For me no problem so I don't care about this message
